Tools/scripts/generate_manifest.py

```diff
@@ -163,6 +164,17 @@ def git_sha_from_git_version(self, filepath):
                 "filepath (%s) does not contain a git sha" % (filepath,))
         return m.group("sha")
 
+    def fwversion_from_git_version(self, filepath):
+        '''parses get-version.txt (as emitted by build_binaries.py, returns
+        git sha from it'''
+        content = open(filepath).read()
+        sha_regex = re.compile("APMVERSION: \S+\s+(\S+)")
```

Comment:
What is this "APM" you speak of?

Reply:
:-)  It's present in vast numbers of pre-existing files on our build server - short of rewriting all those files we're kind of stuck with this ATM.  It was like that when I got here (pretty sure, anyway!)

We could consider understanding e.g. `ARDUPILOT_VERSION: ` as well and add another line into those files - but I don't know if there are external tools also looking for that string, soooo.....

Reply:
>It was like that when I got here

LMAO